src/functions/auth.ts

```diff
@@ -0,0 +1,19 @@
+import firebase from "firebase/app";
+import { accessTokenKey } from "utils/localStorage";
+
+type CheckAccessTypes = {
+  venueId: string;
+  password?: string;
+  email?: string;
+  code?: string;
+  token?: string;
+};
+
+export const checkAccess = async (data: CheckAccessTypes) =>
+  await firebase.functions().httpsCallable("access-checkAccess")(data);
+
+export const setLocalStorageToken = (venueId: string, token: string) =>
+  localStorage.setItem(accessTokenKey(venueId), token);
+
+export const getLocalStorageToken = (venueId: string) =>
+  localStorage.getItem(accessTokenKey(venueId));
```

Comment:
These feel like they would make more sense being in `src/utils/localStorage.ts` alongside `accessTokenKey`